src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -151,6 +146,9 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
         this._columns[MutationTableColumnType.COPY_NUM].shouldExclude = ()=>{
             return (!this.props.discreteCNAMolecularProfileId) || (this.getSamples().length > 1);
         };
+        this._columns[MutationTableColumnType.FUNCTIONAL_IMPACT].shouldExclude = () => {
+            return !this.props.enableFunctionalImpact;
```

Comment:
This is where we decide the fate of the column :)

Reply:
Actually it might be even better to handle the `shouldExclude` condition in the parent class (`MutationTable`)